static/src/stylesheets/module/nav/_navigation.scss

```diff
@@ -406,6 +412,26 @@ $c-navigation-expandable-background: colour(neutral-1);
         padding-bottom: $gs-baseline/4;
         padding-left: $gs-gutter/2;
     }
+
+    &.global-navigation__children--history {
+        background: guss-colour(neutral-3, $pasteup-palette);
+
+        .button--tag {
+            margin-top: $gs-baseline / 2;
+            margin-left: $gs-gutter / 4;
+            margin-right: 0;
+            margin-bottom: 0;
+
+          &:hover {
+             text-decoration: none;
+          }
+        }
+        @include mq($until: mobileLandscape) {
+            & {
```

Comment:
I'd probably also move it above `.button--tag` so it's closer to the the actual selector. Maybe.